pymeasure/display/Qt.py

```diff
@@ -52,9 +52,11 @@ def fromUi(*args, **kwargs):
 def qt_min_version(major, minor=0):
```

Comment:
It seems to me that this function is not used anymore now you've removed the calls in `inputs.py`. Is there a reason to keep the function around? Maybe for backwards compatilibity, though I do not see the point in this case, since it is not a real feature of pymeasure; what do you think @bilderbuchi?

Reply:
Yeah I think it's fine to remove it if we don't use it in the code base.

Reply:
OK, I will do this then.